index.bs

```diff
@@ -281,6 +281,40 @@ In these cases, features should be designed such that it is possible for
 code to gracefully fail or be polyfilled.
 See [[#feature-detect]].
 
+<h3 id=new-features>New features can add functionality to the Web</h3>
+
+A new Web feature can add capabilities to the Web with little concern about the effect on existing content.
+The Web includes many extension points that allow for additions;
+see for example [[HTML#extensibility]].
+
+Before adding items, consider integration with existing, similar capabilities.
+If this leads to a preferred design approach that cannot be implemented by only adding items,
+it might still be possible; see [[#removing-features]].
+
+Do not assume that a change or removal is impossible without first checking.
+Before adding items, consider integration with existing, similar capabilities.
+If this leads to a preferred design approach that cannot be implemented by only adding items,
+it might still be possible; see [[#removing-features]].
+Do not assume that a change or removal is impossible without first checking.
+
+<h3 id=removing-features>Remove or change capabilities only once you understand existing usage</h3>
+
+Stability is a high priority goal of the Web.
```

Comment:
```suggestion
Take care when adding or changing functionality prioritize the stability of the web platform.
```

Reply:
Not so sure about this.  Maybe "Compatibility with existing content" instead of "stability"?

Reply:
```suggestion
Compatibility with existing content is an important goal of the Web.
```